.github/workflows/create-quantic-package.yml

```diff
@@ -59,19 +58,8 @@ jobs:
       - name: Copy static resources
         working-directory: ./packages/quantic
         run: npm run copy:staticresources
-      - name: Connect to DevHub organization
-        working-directory: ./packages/quantic
-        run: |
-          echo "$SFDX_AUTH_JWT_KEY" > server.key
-          sfdx force:auth:jwt:grant --clientid $SFDX_AUTH_CLIENT_ID --jwtkeyfile server.key --username $SFDX_AUTH_JWT_USERNAME --setdefaultdevhubusername
-      - name: Remove translation files
-        run: |
-          cd packages/quantic/force-app/main/translations
-          rm *.translation-meta.xml
       - name: Create package version
         working-directory: ./packages/quantic
         run: |
-          sfdx force:package:version:create --package $PACKAGE_ID --installationkeybypass --codecoverage --wait 30 --json > package-version-result.json
-          cat package-version-result.json
-          PACKAGE_VERSION_ID="$(cat package-version-result.json | jq '.result.SubscriberPackageVersionId' -r)"
-          echo $PACKAGE_VERSION_ID
\ No newline at end of file
+          echo "$SFDX_AUTH_JWT_KEY" > server.key
+          node_modules/.bin/ts-node scripts/build/create-package.ts --remove-translations
```

Comment:
This isn't new fyi we already do that on the daily build and did it for v1.0.0 for the following reason:
If we package the translations then Translation Workbench because a requirement to use the package. However the translation workbench feature in salesforce is not available in every type of org and not enabled by default. So the plan so far is to package without the translations. Once we officially support many languages we can start publishing two packages in parallel. One "international" one and one "english" one.

Reply:
Ah ok, right. Thanks.

Reply:
My pleasure